lib/node_modules/@stdlib/math/base/assert/is-prime/src/main.c

```diff
@@ -0,0 +1,139 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/assert/is_prime.h"
+#include "stdlib/math/base/special/sqrt.h"
+#include "stdlib/math/base/special/floor.h"
+#include "stdlib/constants/float64/max_safe_integer.h"
+#include <stdint.h>
+#include <math.h>
+
+static const double WHEEL_PRIMES[45] = {
+    11.0, 13.0, 17.0, 19.0, 23.0, 29.0, 31.0, 37.0, 41.0, 43.0, 47.0, 53.0, 59.0, 61.0, 67.0, 71.0, 73.0, 79.0, 83.0, 89.0, 97.0,
+    101.0, 103.0, 107.0, 109.0, 113.0, 127.0, 131.0, 137.0, 139.0, 149.0, 151.0, 157.0, 163.0, 167.0, 173.0, 179.0, 181.0,
+    191.0, 193.0, 197.0, 199.0, 211.0
+};
+
+/**
+* Test if a finite double-precision floating-point number is a prime number.
+*
+* @param x    input value
+* @return	  output value
+*
+* @example
+* #include <stdbool.h>
+*
+* bool out = stdlib_base_is_prime( 3.0 );
+* // returns true
+*/
+bool stdlib_base_is_prime( const double x ) {
+	double N;
+	int32_t i;
```

Comment:
```suggestion
	int32_t i;
	double N;
```

Order declarations by length.